docs/cmd/kn_service_update.md

```diff
@@ -38,28 +38,31 @@ kn service update NAME [flags]
-      --tag strings              Set tag (format: --tag revisionRef=tagName) where revisionRef can be a revision or '@latest' string representing latest ready revision. This flag can be specified multiple times.
-      --traffic strings          Set traffic distribution (format: --traffic revisionRef=percent) where revisionRef can be a revision or a tag or '@latest' string representing latest ready revision. This flag can be given multiple times with percent summing up to 100%.
-      --untag strings            Untag revision (format: --untag tagName). This flag can be spcified multiple times.
-      --wait-timeout int         Seconds to wait before giving up on waiting for service to be ready. (default 60)
+      --async                      Update service and don't wait for it to become ready.
+      --concurrency-limit int      Hard Limit of concurrent requests to be processed by a single replica.
+      --concurrency-target int     Recommendation for when to scale up based on the concurrent number of incoming request. Defaults to --concurrency-limit when given.
+  -e, --env stringArray            Environment variable to set. NAME=value; you may provide this flag any number of times to set multiple environment variables. To unset, specify the environment variable name followed by a "-" (e.g., NAME-).
+      --env-from stringArray       Config a envfrom with a config map or secret. (config-map | secret):CONFIG_MAP_OR_SECRET_NAME you may provide this flag any number of times. To unset, specify the config map name followed by a "-" (e.g., config-map:CONFIG_MAP_NAME- or secret:SECRET_NAME-).
+  -h, --help                       help for update
+      --image string               Image to run.
+  -l, --label stringArray          Service label to set. name=value; you may provide this flag any number of times to set multiple labels. To unset, specify the label name followed by a "-" (e.g., name-).
+      --limits-cpu string          The limits on the requested CPU (e.g., 1000m).
+      --limits-memory string       The limits on the requested memory (e.g., 1024Mi).
+      --lock-to-digest             keep the running image for the service constant when not explicitly specifying the image. (--no-lock-to-digest pulls the image tag afresh with each new revision) (default true)
+      --max-scale int              Maximal number of replicas.
+      --min-scale int              Minimal number of replicas.
+  -n, --namespace string           List the requested object(s) in given namespace.
+      --no-lock-to-digest          do not keep the running image for the service constant when not explicitly specifying the image. (--no-lock-to-digest pulls the image tag afresh with each new revision)
```

Comment:
Nit ... Start with capital to be consistent with others "Do not ..."

Reply:
Unfortunately, it is not written by me. BTW, "do not" is generated by a library (Corbra?).